testing/browserpass/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=browserpass
+pkgver=3.0.6
+pkgrel=0
+pkgdesc="Browser extension for pass"
+url="https://github.com/browserpass/browserpass-native"
+arch="all"
+license="MIT"
+depends="pass"
+makedepends="go git"
+options="!check" # no tests to be run
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/browserpass/browserpass-native/archive/$pkgver.tar.gz"
+builddir="$srcdir/github.com/browserpass/browserpass-native"
+
+prepare() {
+	mkdir -p "${builddir%/*}"
+	mv "$srcdir/browserpass-native-$pkgver" "$builddir"
+	default_prepare
+	sed -i "s:%%replace%%:/usr/bin/browserpass-native:" \
+			browser-files/chromium-host.json browser-files/firefox-host.json
+}
+
+build() {
+	export GOPATH="$startdir"
+	go get -v -d
+	go fix
+	go build
```

Comment:
Just curious, why don't you use `make` to run all these commands, [like Arch Linux package does](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/browserpass)?

Reply:
Fixed now. Didn't look at it to be frank since the Makefile didn't work for me with browserpass 2.x on Exherbo - thanks for the heads up, it works now :)

Reply:
@Cogitri Do you still need to push this?

Reply:
Oh right! Thanks for noticing? :)

Reply:
Pushed new version.